logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionStatusMapper.kt

```diff
@@ -9,7 +9,7 @@ interface ConnectionStatusMapper {
     fun fromDaoToApi(state: ConnectionEntity.State): ConnectionStateDTO?
     fun fromApiModel(state: ConnectionStateDTO): ConnectionState
     fun toApiModel(state: ConnectionState): ConnectionStateDTO?
-    fun fromDaoModel(state: ConnectionEntity.State): ConnectionState
+    fun fromDaoModel(state: ConnectionEntity.State?): ConnectionState
```

Comment:
I'm not a fan of allowing optional arguments. The caller handle case the when the argument is null.